source/fuzz/fuzzer.h

```diff
@@ -83,8 +75,13 @@ class Fuzzer {
   // returns a successful result status together with the transformed binary and
   // the sequence of transformations that were applied.  Otherwise, returns an
   // appropriate result status together with an empty binary and empty
-  // transformation sequence.
-  FuzzerResult Run();
+  // transformation sequence. |num_of_transformations| is equal to the maximum
+  // number of transformations applied in a single call to this method. This
+  // parameter is ignored if its value is equal to 0.
```

Comment:
Nit: it's likely that |num_of_transformations| will be slightly exceeded, since we only consider stopping once a fuzzer pass is done. So maybe add a brief remark after this: "Because fuzzing cannot stop mid way through a fuzzer pass, fuzzing will stop after the fuzzer pass that exceeds |num_of_transformations| has completed, so that the total number of transformations may be somewhat larger than this number."

Reply:
Fixed in 5967f1f.